packages/manager/apps/hub/src/e2eTests/hubHome/CA_EU_US_hub.e2e.js

```diff
@@ -0,0 +1,60 @@
+import config from '../../../../../tools/testcafe/config';
+import {
+  userRole,
+  userRoleDisconnect,
+} from '../../../../../tools/testcafe/roles';
+import navbar from '../../../../../tools/testcafe/pages/components/navbar';
+import HubPage from '../../../../../tools/testcafe/pages/hub/hubPage';
+import ServersListDashboard from '../../../../../tools/testcafe/pages/hub/dedicatedServerDashboard';
+
+const user = userRole(config);
+
+fixture('check hub page')
+  .meta({
+    service: [
+      'ovh.com-manager',
+      'ca.ovh.com-manager',
+      'us.ovhcloud.com-manager',
+    ],
+    type: 'regression',
+    severity: 'critical',
+    priority: 'high',
+    scope: 'hub',
+  })
+  .page(config.baseUrl)
+  .beforeEach(async (t) => {
+    await t.useRole(user);
+  });
+
+test('it should access to Manager Hub', async (t) => {
+  await t.expect(navbar.navbar.exists).ok();
+}).after(async () => {
+  await userRoleDisconnect(config);
+});
+
+test('confirm link to all orders list', async () => {
+  const hubPage = new HubPage();
+  await hubPage.confirmCurrentPage();
+  await hubPage.confirmOrdersListLink();
+});
```

Comment:
no need to put `await userRoleDisconnect(config);` like above?

Reply:
it's a mistake i should have put it at the end of the last test,

Reply:
only one is needed on a test suite file, after the last test. fixed